### PR TITLE
Adds Jsonp support to Admin Service. Jetty:run build option to admin service as well as resolves issue #259

### PR DESCRIPTION
**Pull request by ebbesson**

### CODE REVIEW

File: admin-service/src/main/java/com/findwise/hydra/admin/rest/JsonpCallbackFilter.java [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+            getResponse().setContentType(getContentType());
+            getResponse().getOutputStream().write(bytes);
+            getResponse().flushBuffer();
+        }
+    }
+
+    private static class JsonpHttpServletRequestWrapper extends HttpServletRequestWrapper {
+
+        private static class ByteArrayServletInputStream extends ServletInputStream {
+
```
Comment (laserval) [inline admin-service/src/main/java/com/findwise/hydra/admin/rest/JsonpCallbackFilter.java]: I don't like the nested internal classes. I'd prefer having all the internal classes here in a separate package.

**Comment by laserval** [comment]
In the future, when updating to upstream, use `git pull --rebase`. This avoids the merge commit, and also helps you see how your changes relate to the changes on upstream. It also makes more sense for pull requests to be only additional commits to merge into master, without the back-and-forth merge commits.

File: admin-service/src/main/java/com/findwise/hydra/admin/rest/JsonpCallbackFilter.java (hunk 1 above)
Comment (Nycander) [inline admin-service/src/main/java/com/findwise/hydra/admin/rest/JsonpCallbackFilter.java]: new package with all of the internal classes as package-private, and this class as public

**Comment by jensbengtsson** [comment]
To be able to support posting in libraries cross domain CORS would need to be implemented right? So a GUI that uses JSONP would only be able to retrieve data, not add?

**Comment by ebbesson** [comment]
Yes, that's correct. Not every browser supports CORS though e.g. IE<10 . 
CORS can be supported in a similar fashion using e.g http://software.dzhuvinov.com/cors-filter.html
Since CORS-artifact is distributed though maven central it should be fine to add it just like a dependency and configure the web-xml

**Comment by laserval** [comment]
Merged this manually, closing. Nice stuff!

**Pull request closed without merging**